aptos/typescript/wallet_adapter_react/src/client/App.tsx

```diff
@@ -0,0 +1,270 @@
+import "./App.css";
+import "@aptos-labs/wallet-adapter-ant-design/dist/index.css";
+import { useState } from "react";
+import axios from 'axios';
+import { 
+  PendingTransactionResponse, 
+  AptosConfig, 
+  Network, 
+  Aptos, 
+  UserTransactionResponse, 
+  SimpleTransaction,
+  Deserializer, 
+  AccountAuthenticator,
+  Hex,
+  MoveString,
+  AccountAddress
+} from "@aptos-labs/ts-sdk";
+import { useWallet } from "@aptos-labs/wallet-adapter-react";
+import { WalletSelector } from "@aptos-labs/wallet-adapter-ant-design";
+
+// Set up an Aptos client for submitting and fetching transactions
+const aptosConfig = new AptosConfig({ network: Network.TESTNET });
+const aptosClient = new Aptos(aptosConfig);
+
+function App() {
+  const {
+    account,
+    signTransaction
+  } = useWallet();
+
+  const currentAccount = account?.address;
+  const [latestDigest, setLatestDigest] = useState<string>();
+  const [latestResult, setLatestResult] = useState<string>();
+  const [newSuccessfulResult, setnewSuccessfulResult] = useState<boolean>();
+  
+
+  // 1. Get the user's input and update the page state.
+  // 2. Build, sponsor, and execute a feePayer SimpleTransaction with the given user input. 
+  //    If there is a connected wallet, represented by `currentAccount` having a value,
+  //    then the connected wallet is the sender. Otherwise, the sender is a backend
+  //    Shinami Invisible Wallet (hard coded for this very simple example app). 
+  const executeTransaction = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setnewSuccessfulResult(false);
+    const form = e.currentTarget
+    const formElements = form.elements as typeof form.elements & {
+      messageText: {value: string}
+    };
+
+    const message = formElements.messageText.value;
+
+    let pendingTxResponse = undefined;
+    try {
+      if (currentAccount) {
+        pendingTxResponse =  await connectedWalletTxFEBuildBESubmit(message, currentAccount);
+                            // await connectedWalletTxBEBuildFESubmit(message, currentAccount);
+                            // await connectedWalletTxBEBuildBESubmit(message, currentAccount);
+                            // await connectedWalletTxFEBuildFESubmit(message, currentAccount);
+      }
+      else {
+        pendingTxResponse = await invisibleWalletTx(message);
+      }
+
+      if(pendingTxResponse?.hash) {
+          waitForTxAndUpdateResult(pendingTxResponse.hash);
+      } else {
+          console.log("Unable to find a digest returned from the backend.");
+      }  
+  } catch (e) {
```

Comment:
Indentation seems off.

Reply:
Fixed

Reply:
Still seems off on the catch block. Can you set up your text editor to do it automatically?

Reply:
I just set up VSCode to format on save and I _think_ it's working now.